plugins/SegmentEditor/templates/_segmentSelector.twig

```diff
@@ -56,10 +56,10 @@
                     </strong></div>
                 {{ 'General_And'|translate }} <div class="auto_archive"><strong>
                         <select class="auto_archive_select">
-                            {% if createRealTimeSegmentsIsEnabled %}
+                            {% if createRealTimeSegmentsIsEnabled or isBrowserArchivingEnabled %}
```

Comment:
@diosmosis not sure this change is actually needed?

Reply:
@tsteur you said:

> I reckon the only remaining change needed is to set in segment editor "pre-processed" as default when browser archiving is disabled.